msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -31,13 +31,13 @@ You can use conversation update events to provide better notifications and more
 
 A bot receives a `conversationUpdate` event in either of the following cases:
 
-* When bot has been added to a conversation.
-* Other members are added to or removed from a conversation.
-* Conversation metadata has changed.
+* When bot is added to a conversation
+* Other members are added to or removed from a conversation
+* Conversation metadata has changed
 
-The `conversationUpdate` event is sent to your bot when it receives information on membership updates for teams where it has been added. It also receives an update when it has been added for the first time for personal conversations.
+ If a new member is added to a channel, the bot receives information with help of `conversationUpdate` event. For more information to add new team members to a channel and receive messages in the same, see [Team members added](#team-members-added) and [Installation update event](#installation-update-event).
 
-The following table shows a list of Teams conversation update events with more details:
+The following table provides a list of Teams conversation update events with more details:
```

Comment:
The table below says members removed event is sent only for groupchat and team, but in fact it is also sent in 1:1, as implied by this note:
![image](https://user-images.githubusercontent.com/767208/169623130-e97944e9-4c57-498e-bfa5-ca869450595e.png)

Reply:
Updated as mentioned.